### ISSUE TITLE
can tflearn DNN model.fit() ops add return info for acc  los valid loss and so on ?

### ISSUE THREAD

**Issue by mlinxiang** [tflearn/tflearn#174]
![image](https://cloud.githubusercontent.com/assets/6283983/16549600/6ede28ec-41d3-11e6-9b9c-786c45cf3d13.png)

i am very glad to use tflearn for my nlp experiment, when i change my tensorflow code to tflearn, after model.fit() and finish training, i found that there is no way to get the train result information but only write to std and summary, but i want to get these info an auto write to my experiment manager system, such as final train loss/ top_k_pred  val_loss val_acc and so on
did already has way to get but i didn't find?

**Comment by aymericdamien** [comment]
This feature is not created yet, but you can easily change the monitor function here:
https://github.com/tflearn/tflearn/blob/master/tflearn/callbacks.py#L107

**Comment by mlinxiang** [comment]
thanks very much, i will try this callbacks function

**Comment by aymericdamien** [comment]
yes, currently you cannot easily add a new one. But you can easily modify the current one.
You can remove these two lines: https://github.com/tflearn/tflearn/blob/master/tflearn/callbacks.py#L193
 193 & 194. This should print the whole lines again and again.

**Comment by litstrong** [comment]
+1. Could pass callback into `fit` function like this:

https://keras.io/callbacks/

**Comment by aymericdamien** [comment]
Yes, that would be nice if someone can update the code to include that future. Otherwise I can update it later.

**Comment by tgallice** [comment]
@aymericdamien I can do it if you want.

**Comment by aymericdamien** [comment]
@tgallice That would be great! Thanks!
